src/borg/archiver.py

```diff
@@ -1313,26 +1313,26 @@ def do_break_lock(self, args, repository):
            If it is compressible, use the C,[L] compression spec given within the
            "auto" specifier. If it is not compressible, use no compression.
 
-       Examples::
+        Examples::
 
-            borg create --compression lz4 REPO::ARCHIVE data
-            borg create --compression zlib REPO::ARCHIVE data
-            borg create --compression zlib,1 REPO::ARCHIVE data
-            borg create --compression auto,lzma,6 REPO::ARCHIVE data
-            borg create --compression-from compression.conf --compression auto,lzma ...
-
-        compression.conf has entries like:
-
-        # example config file for --compression-from option
-        #
-        # Format of non-comment / non-empty lines:
-        # <compression-spec>:<path/filename pattern>
-        # compression-spec is same format as for --compression option
-        # path/filename pattern is same format as for --exclude option
-        none:*.gz
-        none:*.zip
-        none:*.mp3
-        none:*.ogg
+             borg create --compression lz4 REPO::ARCHIVE data
+             borg create --compression zlib REPO::ARCHIVE data
+             borg create --compression zlib,1 REPO::ARCHIVE data
+             borg create --compression auto,lzma,6 REPO::ARCHIVE data
+             borg create --compression-from compression.conf --compression auto,lzma ...
+
+        compression.conf has entries like::
+
+             # example config file for --compression-from option
+             #
+             # Format of non-comment / non-empty lines:
+             # <compression-spec>:<path/filename pattern>
+             # compression-spec is same format as for --compression option
+             # path/filename pattern is same format as for --exclude option
+             none:*.gz
+             none:*.zip
+             none:*.mp3
+             none:*.ogg
```

Comment:
to start, i just copied another borg help section. so if indentation was wrong, it needs to be checked there also.